remove ICS categories from swagger

hschoenburg opened the pull request:
<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺
v                               ✰  Thanks for creating a PR! ✰    
v    Before smashing the submit button please review the checkboxes.
v    If a checkbox is n/a - please still include it but + a little note why
☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->

- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/develop/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry: `clog add [section] [stanza] [message]`
- [ ] rereviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

alexanderbez commented:
@hschoenburg we now commit the generated statik file. Please run: `make update-swagger-docs` and commit the resulting file :)

hschoenburg commented:
Thanks for the comments guys. I added a newly generated statik.go, as well as Auth and Mint tags.  @fedekunze I wasn't completely clear on which `node_*` endpoint was duplicate but I think I removed the right one (with the Misc tag).

hschoenburg commented on client/lcd/swagger-ui/swagger.yaml:
```diff
@@ -30,28 +36,17 @@ paths:
       responses:
         200:
           description: Plaintext version i.e. "v0.25.0"
-  /node_version:
```
not sure if this was the right one to remove

fedekunze replied:
yes, that's the one

fedekunze commented:
`check_statik` test failing

hschoenburg commented:
sorry forgot to fix the statik.go check until now. Lmk if there is anything more I can do here. Looks like the PR needs to be merged locally because of the statik.go conflict.

alexanderbez commented:
There is a conflict @hschoenburg. Perhaps the statik file you generated should be the one committed. How did you generate it?

hschoenburg commented:
> There is a conflict @hschoenburg. Perhaps the statik file you generated should be the one committed. How did you generate it?

@alexanderbez >> thanks for following up. Since this PR makes changes to the swagger.yaml I imagine  the statik.go in this branch is the one that should be merged into master. After committing the changes to swagger.yaml I ran `$ make update-swagger-docs` and committed the new statik.go file.  Was that the right process? The statik check it passing now. But because of the conflict I think this merge needs to be done locally?

alexanderbez commented:
That is the correct process @hschoenburg -- simply commit your `statik.go` file in favor of the one on `master` to resolve the conflicts :)

sabau commented:
Shouldn't we remove also the implementation along with the documentation? Or we end up having those two working but not documented:
- /bank/balances/{address}
- /staking/delegators/{delegatorAddr}/txs

fedekunze commented:
@hschoenburg can we have this for the release next week?

fedekunze commented on client/lcd/swagger-ui/swagger.yaml:
```diff
@@ -320,36 +316,11 @@ paths:
           description: The tx was malformated
         500:
           description: Server internal error
-  /bank/balances/{address}:
```
We decided to not delete this as per discussion with @alessio and @sabau

marbar3778 commented:
> need to resolve conflicts and add `clog` entry

decided to not add a clog entry

marbar3778 commented:
also, the commit message steemed from something else I apologize for it
